apps/dh/app-dh/src/index.html

```diff
@@ -19,7 +19,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>DataHub</title>
+    <title>DataHub!</title>
```

Comment:
❗

Reply:
@ManBearTM it’s not for review yet ;) … I’m just testing stuff